avatars/processors/__init__.py

```diff
@@ -13,5 +16,6 @@
     "PerturbationProcessor",
     "ProportionProcessor",
     "RelativeDifferenceProcessor",
-    "ToCategoricalProcessor ",
+    "ToCategoricalProcessor",
```

Comment:
thanks nice fix

Reply:
> I don't see the added value of the processor

This processor is needed with hierarchical data if we have a variable like `mileage` that can only increase with time. If we leave it as is, we'll have incoherent values after avatarization. This processor expresses the value as a difference with the previous values (in other words and in the car context, the distance of a trip). The postprocess re-expresses it as car mileage.